src/constants/Colors.ts

```diff
@@ -28,4 +28,6 @@ export default {
   connectionLableColor: '#ffffff',
   graphBarColor: 'grey',
   rootViewColor: '#fff',
+  ventOnButtonColor: 'green',
```

Comment:
The names should be `startVentilationButtonColor` and `stopVentilationButton` for consistency